_inc/client/components/jetpack-termination-dialog/features.jsx

```diff
@@ -134,6 +147,17 @@ class JetpackTerminationDialogFeatures extends Component {
 						</ul>
 					</div>
 				) }
+				{ connectedPlugins && (
```

Comment:
Can it happen that `connectedPlugins` is an empty array? Should we account for that here?

Reply:
That's a good catch. Since empty array casts to `true`, the block would show up even if no plugins used the connection.
Fixed now.